### PR TITLE
Feature/lattice 2271 cleanup

### PR DESCRIPTION
**Pull request by geekbeast**

### CODE REVIEW

File: src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt [hunk 1]
```diff
@@ -300,27 +302,24 @@ class PostgresEntityDataQueryService(
...
-                    }
+                    }.toMap()
 
                     tombstoneFn(version, entityBatch)
                     val upc = upsertEntities(
                             entitySetId,
                             entityBatch,
                             authorizedPropertyTypes,
-                            version,
+                            version+1,
```
Comment (katherinebernstein) [inline src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt]: 🤔

File: src/main/kotlin/com/openlattice/data/storage/PostgresDataSinkService.kt [hunk 2]
```diff
@@ -12,6 +12,7 @@ import kotlin.collections.HashMap
 private val logger = LoggerFactory.getLogger(PostgresDataSinkService::class.java)
 
 @Service
+@Deprecated("This needs to be removed")
```
Comment (makosblade) [inline src/main/kotlin/com/openlattice/data/storage/PostgresDataSinkService.kt]: 🎉

File: src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt (hunk 1 above)
Comment (makosblade) [reply src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt]: I too am confused about this. What is this for?

File: src/main/kotlin/com/openlattice/ids/HazelcastIdGenerationService.kt [hunk 3]
```diff
@@ -19,7 +19,7 @@ class HazelcastIdGenerationService(clients: HazelcastClientProvider, private val
      */
     companion object {
         private const val MASK_LENGTH = 16
-        const val NUM_PARTITIONS = 1L shl MASK_LENGTH //65536
+        const val NUM_PARTITIONS = 1 shl MASK_LENGTH //65536
```
Comment (makosblade) [inline src/main/kotlin/com/openlattice/ids/HazelcastIdGenerationService.kt]: nice catch

**Pull request merged**